plugins/logging/pkg/backend/install.go

```diff
@@ -0,0 +1,131 @@
+package backend
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	opnicorev1beta1 "github.com/rancher/opni/apis/core/v1beta1"
+	capabilityv1 "github.com/rancher/opni/pkg/apis/capability/v1"
+	opnicorev1 "github.com/rancher/opni/pkg/apis/core/v1"
+	"github.com/rancher/opni/pkg/capabilities"
+	"github.com/rancher/opni/pkg/capabilities/wellknown"
+	"github.com/rancher/opni/pkg/resources"
+	"github.com/rancher/opni/pkg/storage"
+	"github.com/rancher/opni/pkg/util"
+	"github.com/rancher/opni/plugins/logging/pkg/errors"
+	"google.golang.org/protobuf/types/known/emptypb"
+	corev1 "k8s.io/api/core/v1"
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	opensearchv1 "opensearch.opster.io/api/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+func (b *LoggingBackend) CanInstall(ctx context.Context, _ *emptypb.Empty) (*emptypb.Empty, error) {
```

Comment:
This is never called for agent v2 - this logic should be used, but abstracted away (see metrics ClusterDriver implementation)